api/v1alpha1/meshsync_types_test.go

```diff
@@ -51,6 +51,28 @@ var _ = Describe("The test case for the meshsync CRDs", func() {
 					Name:      "default",
 				},
 			},
+			Config: MeshsyncConfig{
+				ListenerConfigs: map[string]ListenerConfig{
+					"global": {
+						Name:           "meshsync-logstream",
+						PublishTo:      "meshery.meshsync.logs",
```

Comment:
What is the mean of "meshery.meshsync.logs"? It is any user story support this?

Reply:
I may not have enough system knowledge to correctly answer you on that one, but I am following the default configuration [here](https://github.com/meshery/meshsync/blob/master/internal/config/default_config.go)

Reply:
Thanks for clarify this.